Dockerfile

```diff
@@ -1,72 +1,109 @@
-# Build Python app.
-FROM python:3.8-bullseye AS backend
+# We use Debian images because they are considered more stable than the alpine
+# ones becase they use a different C compiler. Debian images also come with
+# all useful packages required for image manipulation out of the box. They
+# however weight a lot, approx. up to 1.5GiB per built image.
+FROM python:3.8 as production
```

Comment:
Image should be ``python:3.8-bullseye``. see: https://github.com/wagtail/wagtail.org/pull/121#discussion_r788967227